src/lib/components/BodyRenderer/Blocks/link.svelte

```
@@ -15,15 +16,22 @@
+    'w-fit relative inline-block bg-accent1-default/15 text-accent1-default  transition-colors hover:bg-accent1-default/25 focus:bg-accent1-default/25';
+
   switch (item.type) {
     case 'link':
       break;
-    case 'navbar':
+    case 'heading-link':
+      classes += ' text-h3-l font-satoshi xs:text-h3-s';
+      break;
+    case 'navbar-link':
       classes += ' text-h3-l font-satoshi xs:text-h3-s';
+      index = item.href.match(/\{routes\.(\w+)\}/)?.[1] || "";
```

ESLint

[error] Strings must use singlequote.

(quotes)